src/main.js

```diff
@@ -29,6 +30,18 @@ if (!app.isPackaged) {
   });
 }
 
+const rpc = new discordrpc.Client({ transport: 'ipc' });
```

Comment:
I would like to see all the discord code (except for the retrieving URL bit in the options, that can be general) grouped behind a feature flag. 
There are a lot of examples, take a look at how I enable the API for example, or shortcut keys).

If you need/want any help do let me know.

Reply:
https://github.com/Mastermindzh/tidal-hifi/blob/master/src/pages/settings/preload.js

This would be a good starting point to figure out what is needed :)

Reply:
That's understandable I will look into the feature flags later today

Reply:
Great. No need to rush things 😄 . I'm happy that you even made a pr!